crates/strategic-server/strategic-stdb-module/src/lib.rs

```diff
@@ -0,0 +1,230 @@
+//! Strategic Layer SpacetimeDB Module (Minimal Demo)
+//!
+//! Simple Mount & Blade style architecture:
+//! - Strategic layer: character, inventory, missions (SpacetimeDB)
+//! - Tactical layer: real-time gameplay (Lightyear server process)
+//!
+//! Flow:
+//! 1. Player clicks "Enter Town" → creates mission with status "pending"
+//! 2. Spawner sees pending mission → starts tactical-server process
+//! 3. Tactical-server writes connection info → status becomes "ready"
+//! 4. Browser client sees "ready" → connects via WebTransport/WebSocket
+//! 5. Mission ends → tactical-server commits results, exits
+
+use spacetimedb::{reducer, table, ReducerContext, Table};
+
+// ============================================================================
+// Tables
+// ============================================================================
+
+/// Character (strategic state only - no HP, that's tactical)
+#[derive(Clone, Debug)]
+#[table(name = character, public)]
+pub struct Character {
+    #[primary_key]
+    pub id: String,
+    pub name: String,
+    pub xp: i32,
+    pub level: i32,
+}
+
+/// Inventory item
+#[derive(Clone, Debug)]
+#[table(name = inventory_item, public)]
+pub struct InventoryItem {
+    #[primary_key]
+    #[auto_inc]
+    pub id: u64,
+    #[index(btree)]
+    pub character_id: String,
+    pub item_id: String,
+    pub qty: i32,
+}
```

Comment:
Do you think that we maybe should split `InventoryItem` to `Inventory` and `Item` tables ? Recent spacetime db introduced joins so I think it should be fine ? Because this seems to glue items to characters pretty hard, but that might not be the case (quest drop isn't associated with a character for example)

And then we can have `Item` as a shared info about the item (e.g. `effects_on_use`, `price`, `damage`, `damage_type`, all kinds of info), and then here we will use `item_id` as `u64` with id of the item in `Item` table.